src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationHandler.cs

```diff
@@ -586,34 +621,13 @@ private string ReadNonceCookie(string nonce)
             return null;
         }
 
-        private AuthenticationProperties GetPropertiesFromState(string state)
+        private AuthenticationProperties SetUserStateOnMessage(OpenIdConnectMessage message, AuthenticationProperties properties)
```

Comment:
This method seems rather trivial and is used only once in the handler. Maybe you can remove it?